interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -4577,19 +4733,20 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
     global $ODIOPTARGET;
     global $OSIOPTARGET;
     global $dated;
+    global $visit_date;
 
     if (!$documents) {
         list($documents) = document_engine($pid);
     }
 
-    $count_OCT = empty($documents['docs_in_name']['OCT']) ? 0 : count($documents['docs_in_name']['OCT']);
+    $count_OCT = count((array)$documents['docs_in_name']['OCT']);
     if ($count_OCT > 0) {
         foreach ($documents['docs_in_name']['OCT'] as $OCT) {
             $OCT_date[] = $OCT['docdate'];
         }
     }
 
-    $count_VF = empty($documents['docs_in_name']['VF']) ? 0 : count($documents['docs_in_name']['VF']);
+    $count_VF = count((array)$documents['docs_in_name']['VF']);
```

Comment:
revert this line since it was addressing a php warning (the uncountable warning)

Reply:
unless your change also doesn't cause the warning, then keep your change